exercises/00-init/src/modules/root/components/header.js

```diff
@@ -1,14 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-class Header extends React.Component {
-    render() {
-      return <h1>{this.props.title}</h1>;
-    }
+function Header (props) {
+      return <h1>{props.title}</h1>;
+}

```

Comment:
It is customary to use arrow functions for function components, i.e. 
```const Header = props => <h1>{ props.title }</h1>;```
or optionally
```const Header = ({ title }) => <h1>{ title }</h1>;```
Not that using a classic function is wrong, but you will never see it in any Salsita project :)

Reply:
As before, I'm defaulting to the syntax I'm most familiar with. If I have to get familiar with a different one, I'll start now, by changing it as you suggest :)